ui/src/overrides.ts

```diff
@@ -213,27 +218,27 @@ export const createThemeOverrides = (
       tdColorModal: cardBackgroundColor,
     },
     Tag: {
-      borderPrimary: `1px solid ${primaryColor}`,
-      textColorPrimary: textColor,
-      colorSuccess: lighten(5),
-      borderSuccess: `1px solid ${lighten(8)}`,
-      textColorSuccess: textColor,
-      closeColorSuccess: textColorSecondary,
-      closeColorHoverSuccess: textColor,
-      closeColorPressedSuccess: darken(5),
-      colorWarning: alpha(0.1, '#FFB020'),
-      borderWarning: `1px solid ${alpha(0.3, '#FFB020')}`,
-      textColorWarning: '#FFB020',
-      closeColorWarning: alpha(0.6, '#FFB020'),
-      closeColorHoverWarning: '#FFB020',
-      closeColorPressedWarning: alpha(0.8, '#FFB020'),
-      color: cardBackgroundColor,
-      textColor: textColorSecondary,
-      border: `1px solid ${borderColor}`,
-      closeColor: textColorSecondary,
-      closeColorHover: textColor,
-      closeColorPressed: darken(5),
-      closeIconColor: alpha(0.8, '#FF0000'),
+      // borderPrimary: `1px solid ${primaryColor}`,
+      // textColorPrimary: textColor,
+      // colorSuccess: lighten(5),
+      // borderSuccess: `1px solid ${lighten(8)}`,
+      // textColorSuccess: textColor,
+      // closeColorSuccess: textColorSecondary,
+      // closeColorHoverSuccess: textColor,
+      // closeColorPressedSuccess: darken(5),
+      // colorWarning: alpha(0.1, '#FFB020'),
+      // borderWarning: `1px solid ${alpha(0.3, '#FFB020')}`,
+      // textColorWarning: '#FFB020',
+      // closeColorWarning: alpha(0.6, '#FFB020'),
+      // closeColorHoverWarning: '#FFB020',
+      // closeColorPressedWarning: alpha(0.8, '#FFB020'),
+      // color: cardBackgroundColor,
+      // textColor: textColorSecondary,
+      // border: `1px solid ${borderColor}`,
+      // closeColor: textColorSecondary,
+      // closeColorHover: textColor,
+      // closeColorPressed: darken(5),
+      // closeIconColor: alpha(0.8, '#FF0000'),
     },
     Upload: {
       peers: {
```

Comment:
The code snippets you've shared seem to be from different components in React (specifically, "List", "Select", "Input", and some styling parts). There doesn't seem to be any irregularity found here that would require a suggestion for optimization. The comments on the code suggest changes like adjusting colors etc., but there seems not to be anything particularly important about them except stylistic.

No specific recommendations exist based on these provided pieces of code, although considering their purpose it's best practice to adhere closely to CSS standards when using variables. So consider doing:

```css
--primary-color: #0d3c8f; /* Adjusted value */
--secondary-color: #b5e2fd;
```
And apply this throughout styles. This is just a general recommendation.